CONTRIBUTING.md

```diff
@@ -1,3 +1,19 @@
+<!--
```

Comment:
I haven't seen many cases where this header is added to markdown files, is there a way to exclude md and/or only include go files?

Reply:
@robscott I can remove the boilerplate for `.md` files. I added this since I noticed some `docs.src/*.md` files contain the boilerplate and others don't. Instead of adding the boilderplate to `.md` files, should it be removed? I'm fine either way as long as we have consistency.

Reply:
Good to know, I didn't realize that we had that header in any markdown files. Taking a look at k/k and k/website it looks like upstream markdown files do not contain the boilerplate. I think it makes sense to follow that precedent here by removing the header from markdown files.